test/Module/Command/DeregisterTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-tooling/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive\Tooling\Module\Command;
+
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
```

Comment:
Is this what allows you to not need to specify `@runInSeparateProcess` for tests that overload classes via Mockery?